src/components/post/Post.jsx

```diff
@@ -169,6 +169,7 @@ class Post extends Component {
       <div className="post-block">
         {this.renderMenu()}
         { post.imageSrc ? <img className="post-img" src={post.imageSrc} alt="more-icon" /> : null }
+        <h5 className="post-title"><em>{post.title}</em></h5>
```

Comment:
Is it possible to decrease the space between the username and the title? It seems to be a lot larger than before. Might be caused by the h5 margin?

Reply:
done.. also fixed an issue where there was no spacing at the top of the post preview dialog.